src/main/kotlin/com/hibob/academy/entity/types.kt

```diff
@@ -1,20 +1,41 @@
 package com.hibob.academy.entity
 
-import java.sql.Timestamp
+import java.sql.Date
 
 data class Pet(
     val id: Long,
     val name: String,
-    val type : String,
-    val dataOfArrival : Timestamp,
+    val type : PetType,
+    val dataOfArrival : Date,
     val companyId : Long
 )
 
 data class Owner(
     val id: Long,
     var name: String?,
     val companyId: Long,
-    val employeeId : Long,
+    val employeeId : String,
     var firstName : String?,
     var lastName : String?
 )
+
+enum class PetType {
+    DOG,
+    CAT,
+    BIRD,
+    FISH,
+    RABBIT;
+
+    companion object {
+        fun fromString(type: String): PetType {
+            return valueOf(type.uppercase())
+        }
+    }
+}
+
+data class PetWithoutType(
+    val id: Long,
+    val name: String,
+    val dataOfArrival : Date,
+    val companyId : Long
+)
```

Comment:
Why do we need this data class? All pets should have type otherwise they are not pets🤷🏻‍♂️